lib/tasks/api_action.rake

```diff
@@ -0,0 +1,11 @@
+namespace :api_action do
+  desc "subdomain analytics and report tasks"
+  
+  task :rerun_failed_actions => [:environment] do 
+    p "starting rerun_failed_actions task  @ #{Time.now}"
```

Comment:
@Pralish we learned a valuable lesson about logging in the last feature release. Nice!